woocommerce/woocommerce
# Entrepreneur Signup: Add Skip button to Design With AI steps.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Add a Skip button to the Design With AI screen when user is entering through the Entrepreneur signup flow.
The Skip button matches the Skip button seen on the signup flow at `wordpress.com/setup/entrepreneur`.





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. You should use `pnpm --filter='@woocommerce/plugin-woocommerce' build` to build the production for tracks event to fire.
1. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store%2Fdesign-with-ai&ref=entrepreneur-signup`.
2. You should see the Skip button on the top right corner.
3. Clicking on the Skip button should:
    - Record a tracks event (open network panel to inspect) called `customize_your_store_entrepreneur_skip_click` with the `step` as prop.
    - Go to My Home



<img width="1177" alt="2024-05-21_23-37-45" src="https://github.com/woocommerce/woocommerce/assets/1287077/37be3a72-0edb-4958-b066-457c30d46cba">
<img width="1182" alt="2024-05-21_23-37-37" src="https://github.com/woocommerce/woocommerce/assets/1287077/8d17ab47-a328-46b9-a1e8-9403cfae740f">
<img width="1182" alt="2024-05-21_23-37-29" src="https://github.com/woocommerce/woocommerce/assets/1287077/30e24f11-109f-40d5-933b-2820c52d9a6b">

### Changelog entry




<details>

#### Significance




#### Type



-   [] Update - Update existing functionality

#### Message 

#### Comment 
Add Skip button to Design With AI steps. (This is internal for Enterpreneur Signup on WPCOM)

</details>
